--- libnet-3.02/Configure.orig	2014-10-11 00:59:32.000000000 +0200
+++ libnet-3.02/Configure	2014-10-27 17:50:29.824984858 +0100
@@ -322,8 +322,7 @@
 
 $msg = 'Enter a list of available NNTP hosts :';
 
-$def = $oldcfg{'nntp_hosts'} ||
-        [ default_hostname($ENV{NNTPSERVER},$ENV{NEWSHOST},'news') ];
+$def = [];
 
 $cfg{'nntp_hosts'} = get_host_list($msg,$def);
 
@@ -331,8 +330,7 @@
 
 $msg = 'Enter a list of available SMTP hosts :';
 
-$def = $oldcfg{'smtp_hosts'} ||
-        [ default_hostname(split(/:/,$ENV{SMTPHOSTS} || ""), 'mailhost') ];
+$def = [];
 
 $cfg{'smtp_hosts'} = get_host_list($msg,$def);
 
@@ -340,7 +338,7 @@
 
 $msg = 'Enter a list of available POP3 hosts :';
 
-$def = $oldcfg{'pop3_hosts'} || [];
+$def = [];
 
 $cfg{'pop3_hosts'} = get_host_list($msg,$def);
 
@@ -348,7 +346,7 @@
 
 $msg = 'Enter a list of available SNPP hosts :';
 
-$def = $oldcfg{'snpp_hosts'} || [];
+$def = [];
 
 $cfg{'snpp_hosts'} = get_host_list($msg,$def);
 
@@ -356,8 +354,7 @@
 
 $msg = 'Enter a list of available PH Hosts   :'  ;
 
-$def = $oldcfg{'ph_hosts'} ||
-        [ default_hostname('dirserv') ];
+$def = [];
 
 $cfg{'ph_hosts'}   =  get_host_list($msg,$def);
 
@@ -365,7 +362,7 @@
 
 $msg = 'Enter a list of available TIME Hosts   :'  ;
 
-$def = $oldcfg{'time_hosts'} || [];
+$def = [];
 
 $cfg{'time_hosts'} = get_host_list($msg,$def);
 
@@ -373,7 +370,7 @@
 
 $msg = 'Enter a list of available DAYTIME Hosts   :'  ;
 
-$def = $oldcfg{'daytime_hosts'} || $oldcfg{'time_hosts'};
+$def = [];
 
 $cfg{'daytime_hosts'} = get_host_list($msg,$def);
 
@@ -437,7 +434,7 @@
  $def = exists $oldcfg{'ftp_firewall_type'}  ? $oldcfg{'ftp_firewall_type'} : 1;
  $ans = Prompt($msg,$def);
  $cfg{'ftp_firewall_type'} = 0+$ans;
- $def = $oldcfg{'ftp_firewall'} || $ENV{FTP_FIREWALL};
+ $def = [];
 
  $cfg{'ftp_firewall'} = get_hostname("FTP proxy hostname :", $def);
 }
